charts/fluent-bit/values.yaml

```diff
@@ -351,8 +352,8 @@ config:
         Daemon Off
         Flush {{ .Values.flush }}
         Log_Level {{ .Values.logLevel }}
-        Parsers_File parsers.conf
-        Parsers_File custom_parsers.conf
+        Parsers_File /fluent-bit/etc/parsers.conf
+        Parsers_File /fluent-bit/etc/conf/custom_parsers.conf
         HTTP_Server On
         HTTP_Listen 0.0.0.0
         HTTP_Port {{ .Values.metricsPort }}
```

Comment:
I'm not a fan of `metricsPort`, really it is the webserver port.

Separately we should probably use Fluent Bit metrics input with Prometheus Exporter output on another dedicated metrics port - then you can have auth for the webserver but not the metrics for example. Or not have webserver, just metrics.

Unfortunately healthcheck runs off the webserver - I'd prefer that split out too so we can have non-auth healthchecks but full secure metrics and webserver.

Reply:
This is the existing pattern (the new charts I'm working on call it the HTTP port). I'm reasonably fine with it being used probes and metrics, for an example of conditional auth Metrics Server can auth part of it's path (although it's almost always disabled with auth handled at the network layer).